src/content/reference/react/lazy.md

```diff
@@ -4,7 +4,7 @@ title: lazy
 
 <Intro>
 
-`lazy` lets you defer loading component's code until it is rendered for the first time.
+`lazy` memungkinkan Anda menunda pemuatan kode komponen hingga komponen tersebut di-*render* untuk pertama kalinya.
```

Comment:
How about this? Continuing from https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5759429

```suggestion
`lazy` memungkinkan Anda menangguhkan (*defer*) pemuatan kode komponen hingga komponen tersebut di-*render* untuk pertama kalinya.
```

Reply:
Applied @mhaidarhanif